docs/riff_streaming_pulsar-gateway.md

```diff
@@ -8,7 +8,10 @@ title: "riff streaming pulsar-gateway"
 
 ### Synopsis
 
-<todo>
+The Pulsar Gateway encapsulates the address of a streaming gateway and a Pulsar Provisioner instance.
+
+The Pulsar Provisioner is responsible for creating topics in a Pulsar cluster.
```

Comment:
Not 100% true (topics are auto-created in pulsar)

Reply:
Copied-pasted from the README https://github.com/projectriff/pulsar-provisioner/ :)

Reply:
We don't need to expose provisioners as a concept